CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## 2017-06-02
```

Comment:
No worries, my fault. 

It's even documented [here](http://keepachangelog.com/en/0.3.0/) but I overlooked it.